### PR TITLE
eslint additions

### PR DESCRIPTION
**Pull request by danielnaab**
- Added env settings for CommonJS/node and ECMAScript 6.
- Added "recommended settings" extends
- Enforce consistency with semi-colons and quotation marks

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -31,6 +37,8 @@
     "no-undef": 2,
     "no-use-before-define": 2,
     "brace-style": [1,"1tbs",{}],
-    "no-mixed-spaces-and-tabs": 2
+    "no-mixed-spaces-and-tabs": 2,
+    "quotes": ["error", "single"],
```
Comment (mbucknell) [inline .eslintrc]: We may have a style fight here. I prefer single quotes. Never mind, I was interpreting this incorrectly. I like single spaces 😄

File: .eslintrc [hunk 2]
```diff
@@ -1,6 +1,12 @@
 {
+  "env": {
+    "commonjs": true,
+    "es6": true
+  },
+  "extends": "eslint:recommended",
```
Comment (ayan-usgs) [inline .eslintrc]: So, I'm wondering if this would allow us to remove a some but not all of the stuff from lines 3 through 33.

Comment (danielnaab) [reply .eslintrc]: Yes - I'll go through them

Comment (ayan-usgs) [reply .eslintrc]: I could also go through them if you would prefer.

Comment (danielnaab) [reply .eslintrc]: No problem, already on it :)

**Pull request merged**